src/oc_erchef/relx.config

```diff
@@ -3,12 +3,7 @@
 %% ex: ts=4 sw=4 ft=erlang et
 {release,{oc_erchef,"1.8.3"},[oc_erchef, syntax_tools, compiler]}.
 {extended_start_script,true}.
-{overlay_vars,"rel/reltool.config"}.
+% Do we need etc/keys? Seems that could live in omnibus?
 {overlay,[{mkdir,"log/sasl"},
           {mkdir,"lib/patches"},
-          {mkdir,"etc/keys"},
-          {copy,"schema","."},
```

Comment:
I think this might break the omnibus build as the schema won't be in the erchef release, where omnibus is expecting it.  If we don't want this in the erchef release anymore, I think we can do this; it just means we also need to update the omnibus build configuration for the schema.